### PR TITLE
[doc] Proofread GGUI.md

### LINKED ISSUE #4691: [Doc] GGUI should already support Mac OS X since v0.9?
image

I believe it supports Mac OS X and the Vulkan backend. The doc needs an update.

### PR DESCRIPTION
**Pull request by Vissidarte-Herman**
Related issue = close #4691

### CODE REVIEW

File: docs/lang/articles/gui/ggui.md [hunk 1]
```diff
@@ -117,34 +132,40 @@ new_color = window.GUI.color_edit_3(name, old_color)
...
 ```
-Call this method at the very end of the render loop for each frame.
 
-## User Input Processing
-To obtain the events that have occurred since the previous poll:
+Call this method *only* at the end of the render loop for each frame.
+
+## User input processing
+
+To retrieve the events that have occurred since the last method call:
```
Comment (Leonz5288) [inline docs/lang/articles/gui/ggui.md:148]: I think since the previous poll is more accurate.

**Review by Leonz5288 — APPROVED** [review]
LGTM

**Review by neozhaoliang — APPROVED** [review]

File: docs/lang/articles/gui/ggui.md [hunk 2]
```diff
@@ -4,41 +4,51 @@ sidebar_position: 2
...
 window = ti.ui.Window('Window Title', (640, 360))
 ```
 
-There are three types of objects that can be displayed on a `ti.ui.Window`:
+The following three types of objects can be displayed on a `ti.ui.Window`:
 
-* 2D Canvas, which can be used to draw simple 2D geometries such as circles, triangles, etc.
-* 3D Scene, which can be used to render 3D meshes and particles, with a configurable camera and light sources.
-* Immediate mode GUI components, e.g., buttons, textboxes, etc.
+- 2D Canvas, which can be used to draw simple 2D geometries such as circles and triangles.
```
Comment (strongoier) [inline docs/lang/articles/gui/ggui.md:35]: These empty lines make spacing weird..

File: docs/lang/articles/gui/ggui.md [hunk 3]
```diff
@@ -117,34 +131,40 @@ new_color = window.GUI.color_edit_3(name, old_color)
...
 ```
 
 Each `event` in `events` is an instance of `ti.ui.Event`. It has the following properties:
-* `event.action`, which could be `ti.ui.PRESS`, `ti.ui.RELEASE`, ...
-* `event.key`, which indicates the key related to this event
 
-To obtain the mouse position:
-* `window.get_cursor_pos()`
+- `event.action`, which can be `ti.ui.PRESS`, `ti.ui.RELEASE`, or `ti.ui.MOTION`.
+
```
Comment (strongoier) [inline docs/lang/articles/gui/ggui.md:156]: another redundant empty line

File: docs/lang/articles/gui/ggui.md [hunk 4]
```diff
@@ -71,42 +79,48 @@ camera.projection_mode(mode)
...
 
-The arguments `vertices`, `indices`, `per_vertex_color`, and `image` are all expected to be Taichi fields. If `per_vertex_color` is provided, `color` will be ignored.
+The arguments `vertices`, `indices`, `per_vertex_color`, and `image` are all expected to be Taichi fields. If `per_vertex_color` is provided, `color` is ignored.
 
 The positions/centers of geometries should be in the world-space coordinates.
 
 :::note
 
-If a mesh has `num` triangles, the `indices` should be a 1D scalar field with a shape of `num * 3` instead of a vector field.
+If a mesh has `num` triangles, the `indices` should be a 1D scalar field with a shape of `num * 3`, *not* a vector field.
```
Comment (ailzhang) [inline docs/lang/articles/gui/ggui.md:107]: I'd prefer this to be length instead of shape, it's easier to understand in the 1D case ;)

**Review by ailzhang — APPROVED** [review]
LGTM!

**Comment by ailzhang** [comment]
Merging since windows CI is not related.

**Pull request merged**